app/controllers/customers_controller.rb

```diff
@@ -142,6 +142,18 @@ def forgot_password
     end
   end
 
+  def reset_token
+    token = params[:token]
+    @customer = Customer.where('token LIKE ?', token).first
+    if @customer.nil? || (Time.now).utc >= (@customer.token_created_at + 10.minutes).utc
```

Comment:
This line needs fixing. Search for uses of `Time` in the app and you will see that `Time.zone.now` is used everywhere rather than `Time.now`.  (You can google it but basically this is so that someday the app can be timezone-aware, and we have to make sure it's done consistently everywhere.)